app/views/content/train-to-be-a-teacher/promos/_events-promo-apply.html.erb

```diff
@@ -1,8 +1,6 @@
 
 <%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
-  <% promo.left_section(
-      heading: "Get Into Teaching events") do %>
-  <% end %>
+  <% promo.left_section(classes: %w[events-background]) %>
   <% promo.right_section(
         caption: "Online and in-person events",
```

Comment:
@ethax-ross according to trello ticket 3870, we need to change the captions in these event promos to a 'span' - can't quite work out how! 🤔 mind taking a look?

Reply:
I'll pick this change up separately as we need to change captions everywhere not just in the promo modules so its better in its own PR